### ISSUE TITLE
Add config-example to install from pip package

### ISSUE THREAD

**Issue by psyclone241** [ctsit/redi#219]
Add config-example to install from pip package. It would be nice if you could install from the pip package and have a config example come with the install. Maybe I'm missing it, but I don't think this currently happens.

**Comment by senrabc** [comment]
Agreed. When you install 0.15.4 now you GET NO EXAMPLE REDI CONFIG.
